docs/GettingStarted.md

```diff
@@ -869,6 +871,29 @@ customer_cache.get(...) # traced call will belong to `customer-cache` service
 invoice_cache.get(...) # traced call will belong to `invoice-cache` service
 ```
 
+### Rest Client
+
+The `rest-client` integration is available through the `ddtrace` middleware:
+
+```ruby
+require 'rest_client'
+require 'ddtrace'
+
+Datadog.configure do |c|
+  c.use :rest_client, service_name: 'rest_client' # global service name
+end
+
+RestClient.get('http://example.com')
```

Comment:
We should add an example of how to reconfigure a Request to be traced differently, using its own settings instead of global ones. This implementation will depend on how our configuration strategy works for this integration. See my other comment for why that might still change.

Reply:
Also, not here, but we should add test examples that demonstrate such configuration works independently of global configuration, without conflict.